mora/service/engagement.py

```diff
@@ -266,28 +281,43 @@ def convert_leave(funcid, effect):
             keys.LEAVE_TYPE: class_cache[get_type_id(effect)],
         }
 
+    def convert_manager(funcid, effect):
+        return {
+            "uuid": funcid,
+
+            keys.PERSON: user_cache[get_employee_id(effect)],
+            keys.ORG_UNIT: unit_cache[get_unit_id(effect)],
+            keys.RESPONSIBILITY: class_cache[get_responsibility(effect)],
+            keys.MANAGER_LEVEL: class_cache[get_manager_level(effect)],
+            keys.MANAGER_TYPE: class_cache[get_type_id(effect)],
+        }
+
     def add_validity(start, end, func):
         func[keys.VALIDITY] = {
             keys.FROM: util.to_iso_time(start),
             keys.TO: util.to_iso_time(end),
         }
         return func
 
+    def get_classes(effect):
+        rels = effect['relationer']
+        return [obj.get('uuid') for obj in itertools.chain(
+            rels.get('opgaver', []),
+            rels.get('organisatoriskfunktionstype', [])
```

Comment:
This is in fact fine, but that's not obvious from reading the code. One might think that e.g. `RESPONSIBILITY_FIELD` refers to something other than, say, `MANAGER_LEVEL_FIELD`, but they do not. All these functions are just begging for a `dict` that configures this…